### ISSUE TITLE
Plugin fails when it encounters .mjs files

### ISSUE THREAD

**Issue by manizm** [openlayers/jsdoc-plugin-typescript#15]
### How to reproduce (simplified):
- Create a `testContainer.mjs` file and create a class `TestContainer`
- Create another file `testService.mjs` file and import test container: 
```
/**
 * @typedef {import('./testContainer.mjs').TestContainer} TestContainer
 */
 
/**
 * @type {TestContainer}
 */
 const myContainer = new TestContainer;
```

### Reason
- Method, `getModuleInfo` creates an absolute path and adds `.js` at the end
- This causes the path to look like: `./testContainer.mjs.js`
- There are other places that uses `moduleId` and it replaces `.js` with empty, also contributes to the issue.

**PS.** I don't mind creating a PR for this, but I have this as blocker right now, so fast reply would be really appreciated

**Comment by ahocevar** [comment]
Please create a pull request. Another way to solve this would be for your project to add `"type": "module"` to package.json and rename your `.mjs` files to `.js`.

**Comment by manizm** [comment]
We cannot add `"type": "module"` because we are slowly adding support for it in an existing large codebase

**Comment by ahocevar** [comment]
Fixed with #16.
